app/components/Headword.tsx

```diff
@@ -22,11 +23,37 @@ const Headword = ({
   isNonCanadian,
   word,
 }: HeadwordProps): JSX.Element => {
+  const params = useParams()
+  const navigate = useNavigate()
+
+  const navigateToEditEntry = useCallback(
+    (e: React.FormEvent<HTMLFormElement>) => {
+      e.preventDefault()
+      navigate(`/entries/${params.headword}/edit`)
+      return false
+    },
+    [navigate, params.headword]
+  )
```

Comment:
So we're using a Callback as opposed to an Action because I can't seem to get the Action working without it throwing a 405 error right now. I feel like that has to do with `Headword.tsx` only being rendered as a member of `Entry` (and `Entry` being used in `Index`).